### PR TITLE
Issue Template Update

### LINKED ISSUE #70: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
**Resources**

**Alternatives / Additional Context**

### PR DESCRIPTION
**Pull request by iArchitSharma**
**Description**

This PR fixes #70

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by thisiskaransgit — COMMENTED** [review]
@iArchitSharma, include the *Contributors Resources* in other templates 
 
📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
:art: Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io/) and [Community Slack](https://slack.layer5.io/).

**Comment by iArchitSharma** [comment]
> @iArchitSharma, include the _Contributors Resources_ in other templates
> 
> 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md). 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9) 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io/) and [Community Slack](https://slack.layer5.io/).

ok sure!

**Comment by iArchitSharma** [comment]
@thisiskaransgit Done!

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ assignees: ''
 - [Meshery documentation site](https://meshery.layer5.io/docs/)
 - [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
 - [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
-
+- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md:19]:
```suggestion
```

**Review by thisiskaransgit — APPROVED** [review]

**Comment by thisiskaransgit** [comment]
Thanks @iArchitSharma 👍🏼

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 1 above)
Comment (iArchitSharma) [reply .github/ISSUE_TEMPLATE/documentation.md]: done!

**Comment by iArchitSharma** [comment]
> Thanks @iArchitSharma 👍🏼

 I'm really glad I could contribute to the project 😊

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**